GliaWidgets/View/Call/CallStyle.swift

```diff
@@ -1,23 +1,80 @@
 import UIKit
 
+/// Style of the call view.
 public class CallStyle: EngagementStyle {
+    /// Title for audio call.
     public var audioTitle: String
+
+    /// Title for video call.
     public var videoTitle: String
+
+    /// Back button style in the header.
     public var backButton: HeaderButtonStyle
+
+    /// Close button style in the header.
     public var closeButton: HeaderButtonStyle
+
+    /// A text to display operator's name. Include `{operatorName}` template parameter in the string to display operator's name.
     public var operatorName: String
+
+    /// Font of the operator's name text.
     public var operatorNameFont: UIFont
+
+    /// Color of the operator's name text.
     public var operatorNameColor: UIColor
+
+    /// Font of the call duration text.
     public var durationFont: UIFont
+
+    /// Color of the call duration text.
     public var durationColor: UIColor
+
+    /// The text shown in the top section of the view.
     public var topText: String
+
+    /// Font of the top text.
     public var topTextFont: UIFont
+
+    /// Color of the top text.
     public var topTextColor: UIColor
+
+    /// The text shown in the bottom section of the view.
     public var bottomText: String
+
+    /// Font of the bottom text.
     public var bottomTextFont: UIFont
+
+    /// Color of the bottom text.
     public var bottomTextColor: UIColor
+
+    /// Style of the button bar.
     public var buttonBar: CallButtonBarStyle
 
+    ///
+    /// - Parameters:
+    ///   - header: Style of the view's header (navigation bar area).
+    ///   - connect: Styles for different engagement connection states.
+    ///   - backgroundColor: View's background color.
+    ///   - endButton: Style of the engagement ending button.
+    ///   - endScreenShareButton: Style of the screen sharing ending button.
+    ///   - preferredStatusBarStyle: Preferred style of the status bar.
+    ///   - audioTitle: Title for audio call.
+    ///   - videoTitle: Title for video call.
+    ///   - backButton: Back button style in the header.
+    ///   - closeButton: Close button style in the header.
+    ///   - operatorName: A text to display operator's name. Include `{operatorName}` template parameter in the string to display operator's name.
```

Comment:
Change here as well if you change it in line 17.

Reply:
Same as above, waiting for a conclusion on that property.

Reply:
Let's see :)